bin/run-e2e-tests.sh

```diff
@@ -0,0 +1,11 @@
+# Exit if any command fails
```

Comment:
Guessing our Windows friends are left out here? 😬

Reply:
Not sure we want to optimize for Windows users here but if dropping this line is enough to make the tests pass there, maybe we should do it?